docs/development.md

```diff
@@ -22,18 +22,23 @@ In this folder, there are 4 items:
 ```json
 {
     "IsLinux" : true
+    // tags: This is a JSON array that describes the tags the image should have:
```

Comment:
if we want to be able to use comments in json we need replace all the `-powershell` with `-pwsh`

Reply:
I'll move it out

Reply:
`- powershell` in Windows PowerShell which did not support JSON comments

Reply:
Either way.